README.md

```diff
@@ -137,17 +185,24 @@ api.unfollow(user_id_to_follow)
   - [ ] 🚧 Threads Tool for LangChain
   - [ ] 📌 Link Threads & LLaMa ([@Mineru98](https://github.com/Mineru98))
   - [ ] 📌 Provide statistical analysis of posts in Threads ([@Mineru98](https://github.com/Mineru98))
-- [x] ✅ Write data (i.e. write automated Threads)
-  - [x] ✅ Create new Thread with text
-    - [x] ✅ Make link previews to get shown
-  - [x] ✅ Create new Thread with media
+- [ ] 🚧 Write data (i.e. write automated Threads)
+  - [ ] 🚧 Create new Thread with text
+    - [ ] 🚧 Make link previews to get shown
+  - [ ] 🚧 Create new Thread with media
   - [ ] 🚧 Create new Thread with multiple images
-  - [x] ✅ Reply to existing Thread
+  - [ ] 🚧 Reply to existing Thread
```

Comment:
Thought we had `api.publish` — correct me if I'm wrong!

Reply:
Yeah but they changed the URL, and every time I tried its a 404 status

Reply:
@drunkleen oh I see, I'll merge this as-is and test/fix in some other PR. 🚀